format.go

```diff
@@ -36,11 +37,25 @@ func disable_colors(v bool) {
 	separator = red(" | ")
 }
 
-func format(text []byte) string {
+func format(text []byte, grep map[string]string) (string, error) {
 	var m map[string]string
 	err := json.Unmarshal(text, &m)
 	if err != nil {
-		return string(text)
+		return "", microerror.Mask(err)
+	}
+
+	if len(grep) > 0 {
+		var notFoundElements []string
+		for k, v := range grep {
+			logVal, exists := m[k]
+			if !exists || logVal != v {
+				notFoundElements = append(notFoundElements, k)
```

Comment:
How about appending `fmt.Sprintf("%s=%q", k, v)`?

Reply:
Sure, I'll change this.